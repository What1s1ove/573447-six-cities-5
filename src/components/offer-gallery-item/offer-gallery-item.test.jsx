import * as React from 'react';
import {render} from 'enzyme';
import OfferGalleryItem from './offer-gallery-item';

describe(`OfferGalleryItem component`, () => {
  it(`should render OfferGalleryItem component`, () => {
    const component = render(
        <OfferGalleryItem imgPath="https://assets.htmlacademy.ru/intensives/javascript-3/avatar/5.jpg" />
    );

    expect(component).toMatchSnapshot();
  });
});
