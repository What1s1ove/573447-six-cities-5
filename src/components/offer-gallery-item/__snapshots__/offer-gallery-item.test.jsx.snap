// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferGalleryItem component should render OfferGalleryItem component 1`] = `
<div
  class="property__image-wrapper"
>
  <img
    alt="Photo studio"
    class="property__image"
    src="https://assets.htmlacademy.ru/intensives/javascript-3/avatar/5.jpg"
  />
</div>
`;
